src/installLogsCollector.d.ts

```diff
@@ -0,0 +1,44 @@
+type Severity = "" | "error" | "warning";
+
+interface SupportOptions {
+    /**
+     * When set to always logs will be printed for successful test as well as failing ones.
+     * @default 'onFail'
+     */
+    printLogs?: "onFail" | "always";
+
+    /**
+     * What types of logs to collect and print.
+     * By default all types are enabled.
+     * The 'cy:command' is the general type that contain all types of commands that are not specially treated.
+     * @default ['cons:log','cons:info', 'cons:warn', 'cons:error', 'cy:log', 'cy:xhr', 'cy:request', 'cy:route', 'cy:command']
+     */
+    collectTypes?: readonly string[];
+
+    /**
+     * Callback to filter logs manually. The type is from the same list as for the collectTypes option.
+     * Severity can be of ['', 'error', 'warning'].
+     * @default undefined
+     */
+    filterLog?:
+    | null
+    | NonNullable<SupportOptions["collectTypes"]>[number]
+    | ((args: [/* type: */ Severity, /* message: */ string, /* severity: */ Severity]) => boolean);
+
+    xhr?: {
+        /**
+         * Whether to print header data for XHR requests.
+         * @default false
+         */
+        printHeaderData?: boolean;
+
+        /**
+         * Whether to print request data for XHR requests besides response data.
+         * @default false
+         */
+        printRequestData?: boolean;
+    };
+}
+
+function installSupport(config?: SupportOptions): void;
```

Comment:
Just to be consistent with the actual name of the functions, please rename in both files.